### PR TITLE
run rustfmt on syntax::parse::lexer

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/libsyntax/parse/lexer/comments.rs [hunk 1]
```diff
@@ -165,19 +162,25 @@ fn consume_whitespace_counting_blank_lines(rdr: &mut StringReader,
...
+                        code_to_the_left: bool,
                         comments: &mut Vec<Comment>) {
     debug!(">>> shebang comment");
     let p = rdr.last_pos;
     debug!("<<< shebang comment");
     comments.push(Comment {
-        style: if code_to_the_left { Trailing } else { Isolated },
-        lines: vec!(rdr.read_one_line_comment()),
-        pos: p
+        style: if code_to_the_left {
```
Comment (sanxiyn) [inline src/libsyntax/parse/lexer/comments.rs]: Bad formatting. (Comment copied from the previous PR.)

File: src/libsyntax/parse/lexer/comments.rs [hunk 2]
```diff
@@ -195,9 +198,13 @@ fn read_line_comments(rdr: &mut StringReader, code_to_the_left: bool,
     debug!("<<< line comments");
     if !lines.is_empty() {
         comments.push(Comment {
-            style: if code_to_the_left { Trailing } else { Isolated },
+            style: if code_to_the_left {
```
Comment (sanxiyn) [inline src/libsyntax/parse/lexer/comments.rs]: Ditto.

File: src/libsyntax/parse/lexer/comments.rs (hunk 1 above)
Comment (tshepang) [inline src/libsyntax/parse/lexer/comments.rs]: I prefer these in separate lines

**Comment by nrc** [comment]
There's a comment missing around line -1279/+1466

**Comment by nrc** [comment]
r+ with the missing comment addressed.

**Comment by tshepang** [comment]
That comment makes me a little confused... why is the shebang considered a comment?

**Comment by nrc** [comment]
You can start a Rust program with a shebang, and it is ignored by Rust, i.e., treated as a comment. Only shells and so forth would process it.

**Pull request merged**